src/Test/intlinear.jl

```diff
@@ -681,6 +681,307 @@ function indicator4_test(model::MOI.ModelLike, config::TestConfig)
     end
 end
 
+function semiconttest(model::MOI.ModelLike, config::TestConfig{T}) where T
+    atol = config.atol
+    rtol = config.rtol
+
+    @test MOIU.supports_default_copy_to(model, #=copy_names=# false)
+    @test MOI.supports(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{T}}())
+    @test MOI.supports(model, MOI.ObjectiveSense())
+    @test MOI.supports_constraint(model, MOI.SingleVariable, MOI.MathOptInterface.Semicontinuous{T})
```

Comment:
MOI.MathOptInterface

Reply:
Thank you! I will change that, the line 843 has the same error.